src/mappings/storage/metadata.ts

```diff
@@ -60,6 +67,9 @@ export async function processStorageOperatorMetadata(
   if (isSet(metadataUpdate.location)) {
     processNodeLocationMetadata(operatorMetadata, metadataUpdate.location)
   }
+  if (isSet(metadataUpdate.operationalStatus)) {
+    processNodeOperationalStatusMetadata('worker', undefined, metadataUpdate.operationalStatus)
```

Comment:
Don't we need to pass the current status as the second argument so worker is not able to override the lead?

Reply:
Good catch. Addressed